### PR TITLE
feat(Icons): Update Delete & Check Icons and add contributing guide GM-63 GM-67

### PR DESCRIPTION
**Pull request by codecaaron**
## Overview
- Adds a new Delete / Cancel icon alternative for Alerts to avoid the `Double X` problem.
- Updates our CheckMark to a handdrawn version.
- Adds contributing guide for adding icons to the Storybook page.

### PR Checklist

- [ ] Related to Abstract designs:
- [x] Related to JIRA ticket: GM-63 GM-67
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.0.2-alpha.3b5e64.0<br/>@codecademy/gamut-labs@1.8.8-alpha.3b5e64.0<br/>@codecademy/gamut@12.2.7-alpha.3b5e64.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2dca77d88d1f08012c0252--gamut-preview.netlify.app>https://5f2dca77d88d1f08012c0252--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2dca77d88d1f08012c0252>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.0.2-alpha.0201d9.0<br/>@codecademy/gamut-labs@1.8.8-alpha.0201d9.0<br/>@codecademy/gamut@12.2.7-alpha.0201d9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2dcb8a3b2742b55cf438f6--gamut-preview.netlify.app>https://5f2dcb8a3b2742b55cf438f6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2dcb8a3b2742b55cf438f6>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/check-icon.svg [hunk 1]
```diff
@@ -1 +1,13 @@
-<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs" viewBox="0 0 140 140" width="140" height="140"><g transform="matrix(5.833333333333333,0,0,5.833333333333333,0,0)"><path d="M23.374.287a1.5,1.5,0,0,0-2.093.345L7.246,20.2,2.561,15.511A1.5,1.5,0,1,0,.439,17.632l5.935,5.934a1.525,1.525,0,0,0,2.279-.186l15.066-21A1.5,1.5,0,0,0,23.374.287Z" fill="#000000" stroke="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="0"></path></g></svg>
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none"
```
Comment (jakemhiller) [inline packages/gamut-icons/src/svg/check-icon.svg:1]: might want to run the formatter on these jawns

Comment (jakemhiller) [reply packages/gamut-icons/src/svg/check-icon.svg:1]: also maybe we should add that to the instructions?

Comment (codecaaron) [reply packages/gamut-icons/src/svg/check-icon.svg:1]: Good call, though I would like to preserve some metadata of the original asset in them (re: slug and containing set if that makes sense)

What do you think about a comment at the top of the file (I think our transformation won't copy it when we convert to react elements)?

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f316dac065f727193160210--gamut-preview.netlify.app>https://5f316dac065f727193160210--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f316dac065f727193160210>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.0.2-alpha.aa1fcd.0<br/>@codecademy/gamut-labs@1.8.8-alpha.aa1fcd.0<br/>@codecademy/gamut@12.2.7-alpha.aa1fcd.0<br/></pre>

**Pull request merged**